docs/node/full-node/full-node-binaries.md

```diff
@@ -49,7 +49,7 @@ This is the easiest way to get started.
    
    | Network      | Node Binary           | Chain Specification File | Chain Info Reference |
    |--------------|-----------------------|--------------------------|----------------------|
-   | Kate Testnet | [v1.6.2-rc1](https://github.com/availproject/avail/releases/tag/v1.6.2-rc1) | [chainspec.raw.json](/configs/kate/chainspec.raw.json) | [Chain Info](/configs/kate/chaininfo.txt) |
+   | Kate Testnet | [v1.6.2-rc1](https://github.com/availproject/avail/releases/tag/v1.6.2-rc1) | [chainspec.raw.json](/kate/chainspec.raw.json) | [Chain Info](/kate/chaininfo.txt) |
```

Comment:
While this is fine for now, I think it is better to point to the explorer URL/chainspec for the latest chainspec as that is easier to update amd remains fairly updated.